pandas/tests/test_series.py

```diff
@@ -3008,6 +3008,27 @@ def test_round(self):
                           name='ts')
         assert_series_equal(result, expected)
         self.assertEqual(result.name, self.ts.name)
+ 
+        s = Series([1.123, 2.123, 3.123], index=lrange(3))               
+        outarr = np.empty(3)
+        result = s.round(out=outarr)
```

Comment:
so drop the ``out`` paramater from ``Series.round`` (you can put that in API changes). We don't support this ANYWHERE else so its really inconsitent.